fairlearn/reductions/_moments/loss.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
```

Comment:
copyright missing. You probably have a whole bunch of `flake8` alerts and this should be one of them.